linode/manager
# M3-5182: RQ-ify Images endpoints, Pt. 1

## Description 📝
The first of two or three PRs related to transitioning Images out of Redux and into React Query.

In terms of functionality, this PR focuses on the Images landing page and the "Capture Image" tab when you go to create an image, and having them rely on RQ instead of Redux.

The next piece in this effort will be the "Upload Image" tab and associated functionality/infrastructure.

## How to test 🧪
Ensure functionality parity of Images landing & the "Capture Image" tab between this branch and prod.

Confirm that the RQ cache is storing and updating the Images data as you would expect. 

## To-Do:
- [x] Fix bug where Image label and descriptions aren't being updated; ensure RQ cache is updated accordingly
- [x] Fix bug where redirection from "Rebuild an Existing Linode" flow doesn't open the modal on Linode Details page 
- [x] Convert `ImagesDrawer.tsx` to a function component
- [x] Fix status bug on landing page after a newly-created Image is ready
